base/io.jl

```diff
@@ -125,6 +125,9 @@ function write(s::IO, a::AbstractArray)
     end
     return nb
 end
+"""Write directly to a named file. Equivalent to `open(io->write(io,x), filename, "w")`."""
```

Comment:
I'd support passing several values as for `write` since it's easy. I'd also follow the docs for `write` and say:
```
    write(filename, x...)

Write the canonical binary representation of a value to file `filename`.
Returns the number of bytes written into the stream.
Equivalent to `open(io->write(io, x...), filename, "w")`.
```